SDKGenerator.njsproj

```diff
@@ -18,7 +18,7 @@
     <ProjectGuid>{2ba807ef-310c-43b5-8d42-1abcfff39be9}</ProjectGuid>
     <ProjectHome>.</ProjectHome>
     <ProjectView>ShowAllFiles</ProjectView>
-    <StartupFile>generate.js</StartupFile>
+    <StartupFile>generate.ts</StartupFile>
```

Comment:
OH, that's different

Reply:
The sln seems to not create the js file properly.
I was experimenting.
VS should understand what .ts files are though, and it built correctly after I manually regenerated the .js files using tsc.